src/sign-verify/clu_x509_verify.c

```diff
@@ -41,10 +42,34 @@ static const struct option verify_options[] = {
 static void wolfCLU_x509VerifyHelp(void)
 {
     WOLFCLU_LOG(WOLFCLU_L0, "./wolfssl verify -CAfile <ca file name> "
-            "[-crl_check] [-partial_chain] <cert to verify>");
+            "[-untrusted <intermidate file> [-crl_check] "
+            "[-partial_chain] <cert to verify>");
```

Comment:
missing bracket with help menu display

```
bash-3.2$ ./wolfssl verify -h
./wolfssl verify -CAfile <ca file name> [-untrusted <intermidate file> [-crl_check] [-partial_chain] <cert to verify>
```

Reply:
OpenSSL handles multiple -untrusted arguments passed in. i.e.:


```
./wolfssl verify -untrusted intermediate1 -untrusted intermediate2 -CAfile cafile leaf-cert
```

 We need to either document only handling a single case or add support for multiple.

Reply:
Enhanced the documentation